### PR TITLE
Doc for the Advanced Audit Trail UI module

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/appstore/modules/advanced-audit-trail-ui.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## 1 Introduction
+
+The [Advanced Audit Trail UI](https://marketplace.mendix.com/link/component/120204) module contains pre-configured snippets, pages, and layout, which help you create search queries and display query results from your Elasticsearch environment inside your own Mendix application. An extensive search interface allows you to create a search query flexibly: inside a time frame (start date time, end date time), by object (code, GUID, entity), by object action (creation, deletion), by user (username, user role), and by snapshot context (additional info, transaction ID, execution ID, stack trace). Additionally, various pre-configured pages display the results of a single object query or a multi-object query.
+
+### 1.1 Dependencies
+
+You should use this module with the [Advanced Audit Trail](https://marketplace.mendix.com/link/component/120237) module.
+
+### 1.2 Solution Package
```
Comment (brfreekmx) [inline content/en/docs/appstore/modules/advanced-audit-trail-ui.md]: Can this paragraph be removed?

File: content/en/docs/appstore/modules/advanced-audit-trail-ui.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+For a working copy, see [Advanced Audit Trail solution package](https://marketplace.mendix.com/link/component/120236).
+
+## 2 Installation
+
+To download and install the Advanced Audit Trail UI module in your app, follow the instructions in the [Importing Content from the App Explorer](/appstore/general/app-store-content/#import) section in *Use Marketplace Content in Studio Pro*. After the app service is installed, you can find it in the **App Explorer**.
```
Comment (brfreekmx) [inline content/en/docs/appstore/modules/advanced-audit-trail-ui.md]: Can the "After the app service is installed" link to the Advanced Audit Trail App Service documentation?

**Review by brfreekmx — CHANGES_REQUESTED** [review]

File: content/en/docs/appstore/modules/advanced-audit-trail-ui.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/modules/advanced-audit-trail-ui.md]: @brfreekmx I removed the paragraph about the solution package.

File: content/en/docs/appstore/modules/advanced-audit-trail-ui.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/modules/advanced-audit-trail-ui.md]: @brfreekmx I think `the app service` here is actually a mistake. It should be changed `the module`. The module here refers to the Advanced Audit Trail UI module.

Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/modules/advanced-audit-trail-ui.md]: @brfreekmx Added Advanced Trail Core to the Installation section as well.

**Comment by Luyao-Zhang-1** [comment]
The UI module doc is moved to the [same PR as the core module](https://github.com/mendix/docs/pull/4443), since the two docs will be released together. I will close this PR.

**Pull request closed without merging**